receiver/windowsperfcountersreceiver/testdata/scraper/no_matching_instance.yaml

```diff
@@ -0,0 +1,7 @@
+resourceMetrics:
```

Comment:
This comparison is ignored in the test itself, but, adding it to document current behavior. This should be updated after #32384 is merged, since no metric will be emitted in this case.

Reply:
Updated via d2876450b707d808515e26175ba5da36ac1949f8